marketplace/google-analytics-app/src/AppConfig.tsx

```diff
@@ -0,0 +1,302 @@
+          contentType.fields = contentType.fields
+            // use only short text fields of content type
+            .filter(f => f.type === 'Symbol')
+            // sort by field name
+            .sort((a, b) => (a.name < b.name ? -1 : a.name > b.name ? 1 : 0));
+          acc[contentType.sys.id] = contentType;
+
+          return acc;
+        }, {}),
+        contentTypes,
+        clientId: savedParams.clientId || '',
+        viewId: savedParams.viewId || ''
+      },
+      () => sdk.app.setReady()
+    );
+
+    app.onConfigure(() => this.configureApp());
+  }
+
+  async configureApp() {
+    const { contentTypes, clientId, viewId } = this.state;
+
+    const EditorInterface = Object.keys(contentTypes).reduce((acc, id) => {
+      acc[id] = { sidebar: { position: 0 } };
+      return acc;
+    }, {});
+
+    return {
+      parameters: {
+        contentTypes,
+        clientId,
+        viewId
+      },
+      targetState: {
+        EditorInterface
+      }
+    };
+  }
+
+  handleChange(name, value) {
+    this.setState({ [name]: value });
+  }
+
+  handleContentTypeChange(prevKey, key) {
+    this.setState(prevState => {
+      const contentTypes = {};
+
+      // remove contentType[prevKey] field and replace with the new contentType
+      // key while preserving key order
+      for (const [prop, value] of Object.entries(prevState.contentTypes)) {
+        contentTypes[prop === prevKey ? key : prop] = value;
+      }
+
+      return {
+        contentTypes
+      };
+    });
+  }
+
+  handleContentTypeFieldChange(key, field, value) {
+    this.setState(prevState => {
+      const prevContentTypes = prevState.contentTypes;
+      const curContentTypeProps = prevContentTypes[key];
+
+      return {
+        contentTypes: {
+          ...prevState.contentTypes,
+          [key]: {
+            ...curContentTypeProps,
+            [field]: value
+          }
+        }
+      };
+    });
+  }
+
+  addContentType() {
+    this.setState(prevState => ({
+      ...prevState,
+      contentTypes: {
+        ...prevState.contentTypes,
+        '': { slugField: '', urlPrefix: '' }
+      }
+    }));
+  }
+
+  removeContentType(key) {
+    this.setState(prevState => {
+      const contentTypes = { ...prevState.contentTypes };
+
+      delete contentTypes[key];
+
+      return { contentTypes };
+    });
+  }
+
+  render() {
+    const { contentTypes, allContentTypes } = this.state;
+
+    return (
+      <>
+        <div className={styles.background} />
+        <div className={styles.body}>
+          <div className={styles.section}>
+            <Typography>
+              <Heading className={styles.spaced}>About Google Analytics</Heading>
+
+              <Paragraph>
+                This app allows you to view pageview analytics of a Contentful entry in the editor
+                sidebar. For installation instructions, please refer to the app&apos;s
+                <a href>documentation</a>.
```

Comment:
Pretty safe bet the URL will be:

```suggestion
                <a href="https://www.contentful.com/developers/docs/extensibility/apps/google-analytics/">documentation</a>.
```